applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -1793,4 +1793,4 @@ send_mwi_update(New, Saved, BoxNumber, Call) ->
                | kz_api:default_headers(?APP_NAME, ?APP_VERSION)
               ],
     lager:debug("updating MWI for vmbox ~s@~s (~b/~b)", [BoxNumber, Realm, New, Saved]),
-    kz_amqp_worker:cast(Command, fun kapi_presence:publish_mwi_update/1).
+    kapps_util:amqp_pool_send(Command, fun kapi_presence:publish_mwi_update/1).
```

Comment:
i would prefer changing in `cf_util`. `kz_amqp_worker` has more options then `kapps_util:amqp..` .
don't get me wrong, `cast` is ok but `call` and `call_collect` are much more flexible in `kz_amqp_worker` and the kapps_util:amqp_...` just calls kz_amqp_worker so we better use it directly.